imgui.go

```diff
@@ -670,9 +670,9 @@ func TreePop() {
 	C.iggTreePop()
 }
 
-// SetNextTreeNodeOpen sets the open/collapsed state of the following tree node.
-func SetNextTreeNodeOpen(open bool, cond Condition) {
-	C.iggSetNextTreeNodeOpen(castBool(open), C.int(cond))
+// SetNextItemOpen sets the open/collapsed state of the following tree node.
+func SetNextItemOpen(open bool, cond Condition) {
+	C.iggSetNextItemOpen(castBool(open), C.int(cond))
```

Comment:
Removing a function would require a major version bump, which I'd like to avoid if possible.
Recommended: Keep `SetNextTreeNodeOpen()` and comment it as "deprecated", which IDEs tend to point out. Have it then call the new, recommended function. This then is identical to the compatibility mode in imgui.h .